src/internal/m365/collection/groups/handlers.go

```diff
@@ -16,6 +16,7 @@ type BackupHandler interface {
 	) (models.Channelable, error)
 	NewChannelsPager(
 		teamID string,
+		fields []string,
```

Comment:
Field selection shouldn't be exposed to any layer above the api, as that conflicts with producing standardized data.  Unless, of course, there's a need to create multiple instances of pagers, each with different fields.  That doesn't seem to me like a pattern we want to take on.  It's much safer and simpler to assume that pagers can be treated like singletons (ie, we'd only ever need to construct one) than to let them be stateful.  If we really need per-enumeration field selection, that's better of handled by some func that accepts a standard pager as part of its properties.

Reply:
Make sense. Updated the code.